kytos-init.sh

```diff
@@ -7,14 +7,16 @@ set -x
 sed -i 's/STATS_INTERVAL = 60/STATS_INTERVAL = 3/g' /var/lib/kytos/napps/kytos/of_core/settings.py
 sed -i 's/LINK_UP_TIMER = 10/LINK_UP_TIMER = 1/g' /var/lib/kytos/napps/kytos/topology/settings.py
 sed -i 's/DEPLOY_EVCS_INTERVAL = 60/DEPLOY_EVCS_INTERVAL = 5/g' /var/lib/kytos/napps/kytos/mef_eline/settings.py
+sed -i 's/BOX_RESTORE_TIMER = 0.1/BOX_RESTORE_TIMER = 0.5/' /var/lib/kytos/napps/kytos/flow_manager/settings.py
+sed -i 's/COOKIE_PREFIX = 0xbb/COOKIE_PREFIX = 0xab/g' /var/lib/kytos/napps/kytos/of_lldp/settings.py
```

Comment:
[`of_lldp PR #33`](https://github.com/kytos-ng/of_lldp/pull/33/files) has landed with `COOKIE_PREFIX = 0xab`, no harm in this sed, but I supposed it could be removed, right @italovalcy?

Reply:
Thanks for point that out, @viniarck. I've just pushed commit 7333b92 addressing this.